src/libraries/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.cs

```diff
@@ -145,6 +145,7 @@ public enum EventOpcode
         Send = 9,
         Receive = 240,
     }
+    [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.All)]
```

Comment:
Do we really need `All` - is that because of the NestedTypes issue?
(we should fix that then - it's been agreed that we should change the behavior of that already).